### ISSUE TITLE
postgres12 throws `duplicate key value violates unique constraint "email_hash_pkey"` messages

### ISSUE THREAD

**Issue by ghost** [go-gitea/gitea#12287]
- Gitea version (or commit ref): 1.12.2
- Operating system: Ubuntu
- Database (use `[x]`):
  - [x] PostgreSQL

## Description

postgres12 throws `duplicate key value violates unique constraint "email_hash_pkey"` messages.

I am not sure when these messages started to appear, but it seems right after the upgrade from `1.12.1` to `1.12.2`.

## Logs

```
...
...
postgres12_1        | 2020-07-21 14:34:26.727 UTC [962] ERROR:  duplicate key value violates unique constraint "email_hash_pkey"
postgres12_1        | 2020-07-21 14:34:26.727 UTC [962] DETAIL:  Key (hash)=(494e4122a76b31b98c7f37243f9bb30c) already exists.
postgres12_1        | 2020-07-21 14:34:26.727 UTC [962] STATEMENT:  INSERT INTO "email_hash" ("hash","email") VALUES ($1, $2)
postgres12_1        | 2020-07-21 14:34:34.354 UTC [986] ERROR:  duplicate key value violates unique constraint "email_hash_pkey"
postgres12_1        | 2020-07-21 14:34:34.354 UTC [986] DETAIL:  Key (hash)=(3d7a739ecd1fd5a0e6d638118d47c699) already exists.
postgres12_1        | 2020-07-21 14:34:34.354 UTC [986] STATEMENT:  INSERT INTO "email_hash" ("hash","email") VALUES ($1, $2)
postgres12_1        | 2020-07-21 14:34:36.073 UTC [997] ERROR:  duplicate key value violates unique constraint "email_hash_pkey"
postgres12_1        | 2020-07-21 14:34:36.073 UTC [997] DETAIL:  Key (hash)=(7c2d0cb25b44568819e1a0e04501aa97) already exists.
postgres12_1        | 2020-07-21 14:34:36.073 UTC [997] STATEMENT:  INSERT INTO "email_hash" ("hash","email") VALUES ($1, $2)
```

**Comment by lunny** [comment]
So this should be fixed after v1.12.2

**Comment by SuperSandro2000** [comment]
I am on 1.12.3 and my log still gets spammed with this.
